src/Maestro/Client/src/MaestroApi.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.DotNet.Maestro.Client
+{
+    partial class MaestroApiResponseClassifier
+    {
+        public override bool IsRetriableException(Exception exception)
+        {
+            return base.IsRetriableException(exception) ||
+                   exception is TaskCanceledException ||
+                   exception is OperationCanceledException ||
+                   exception is HttpRequestException ||
+                   exception is RestApiException raex && raex.Response.Status >= 500 && raex.Response.Status <= 599 ||
+                   exception is IOException ||
+                   exception is SocketException ||
+                   exception is RestApiException jobListEx && jobListEx.Response.Status == 400 && jobListEx.Message.Contains("Provided Job List Uri is not accessible");
```

Comment:
This condition is specific to the Helix Api. It can be removed for maestro.

Reply:
okay removing condition and keeping the override method only.